.buildkite/scripts/run-tests.ps1

```diff
@@ -0,0 +1,46 @@
+$ErrorActionPreference = "Stop" # set -e
+# Forcing to checkout again all the files with a correct autocrlf.
+# Doing this here because we cannot set git clone options before.
+function fixCRLF {
+    Write-Host "-- Fixing CRLF in git checkout --"
+    git config core.autocrlf input
+    git rm --quiet --cached -r .
+    git reset --quiet --hard
+}
+
+function withGolang($version) {
+    Write-Host "-- Install golang --"
+    choco install -y golang --version $version
+    $env:ChocolateyInstall = Convert-Path "$((Get-Command choco).Path)\..\.."
+    Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
+    refreshenv
+    go version
+    go env
+}
+
+function withGoJUnitReport {
+    Write-Host "-- Install go-junit-report --"
+    go install github.com/jstemmer/go-junit-report/v2@latest
+}
+
+function withMage($version) {
+    Write-Host "-- Install Mage --"
+    go mod download -x
+    go install github.com/magefile/mage@v$version
+}
+
+fixCRLF
+withGolang $env:SETUP_GOLANG_VERSION
+withMage $env:SETUP_MAGE_VERSION
+withGoJUnitReport
+
+$ErrorActionPreference = "Continue" # set +e
+mage -debug test > test-report.txt
```

Comment:
Do we need the `-debug`?

Reply:
@alexsapran in the Jenkins job we call it with `debug`, I can remove it, just decided do the same since I'm not sure which problems can occur with Mage